### PR TITLE
Firefox

### PR DESCRIPTION
**Pull request by christiandrey**

### CODE REVIEW

File: src/background/background.html [hunk 1]
```diff
@@ -1,6 +1,4 @@
 <html>
   <head></head>
-  <body>
-    <script src="https://ssl.google-analytics.com/analytics.js" async></script>
```
Comment (kelonye) [inline src/background/background.html:4]: Same...

File: src/manifest.firefox.json [hunk 2]
```diff
@@ -89,7 +95,7 @@
   "background": {
     "page": "background.html"
   },
-  "content_security_policy": "script-src 'self' https://ssl.google-analytics.com; object-src 'self'",
```
Comment (kelonye) [inline src/manifest.firefox.json:92]: Same...

File: src/background/analytics.js [hunk 3]
```diff
@@ -1,23 +1,42 @@
-function createQueueFunction() {
```
Comment (kelonye) [inline src/background/analytics.js:1]: Revert this..

File: src/manifest.firefox.json [hunk 4]
```diff
@@ -2,13 +2,19 @@
   "name": "Toggl Plan: Project Planning Calendar",
   "version": "2.0.4",
   "manifest_version": 2,
+  "author": "Toggl <support@toggl.com>",
```
Comment (kelonye) [inline src/manifest.firefox.json:5]: Looks like this is really the only addition needed :-D

Comment (christiandrey) [reply src/manifest.firefox.json:5]: > Looks like this is really the only addition needed :-D

How about we discard this branch altogether and just add this in a new branch? Just to be sure that everything works fine.

**Comment by kelonye** [comment]
Awesome

**Pull request closed without merging**